### PR TITLE
add basic alerting for Kueue

### PR DESCRIPTION
**Pull request by KPostOffice**
depends on: https://github.com/red-hat-data-services/kueue/pull/3

ref: https://issues.redhat.com/browse/RHOAIENG-6819

### CODE REVIEW

File: config/monitoring/prometheus/apps/prometheus-configs.yaml [hunk 1]
```diff
@@ -1240,6 +1259,20 @@ data:
...
+    - name: Distributed Workloads Kueue
+      interval: 1m
+      rules:
+        - alert: Kueue Operator is not running
+          expr: absent(up{job=~'Kueue Operator'}) or up{job=~'Kueue Operator'} != 1
+          labels:
+            severity: warning
+          annotations:
+            description: This alert fires when the Kueue Operator is not running.
+            triage: 'https://gitlab.cee.redhat.com/service/managed-tenants-sops/-/blob/main/RHODS/Distributed-Workloads/kuberay-operator-availability.md'  # TODO: what link here?
```
Comment (zdtsw) [inline config/monitoring/prometheus/apps/prometheus-configs.yaml:1273]: i think you will need to create a new file for kueue in **https://gitlab.cee.redhat.com/service/managed-tenants-sops/-/blob/main/RHODS/Distributed-Workloads** and link it here.

Comment (zdtsw) [reply config/monitoring/prometheus/apps/prometheus-configs.yaml:1273]: as long as this is updated, i am fine with this PR

Comment (zdtsw) [inline config/monitoring/prometheus/apps/prometheus-configs.yaml:1273]:
```suggestion
            triage: 'https://gitlab.cee.redhat.com/service/managed-tenants-sops/-/blob/main/RHODS/Distributed-Workloads/kueue-operator-availability.md'
```

**Comment by zdtsw** [comment]
once this is in "main" we need backport to rhoai-2.9

**Comment by KPostOffice** [comment]
This should all be g2g now. The SOPs PR has been merged

**Review by zdtsw — APPROVED** [review]

**Pull request merged**